scripts/e2e/bootstrap_job/spec/provider-components-v2.0.template

```diff
@@ -0,0 +1,960 @@
+          type: object
+        vsphereMachine:
+          type: string
+      required:
+      - vsphereMachine
+      - machineVariables
+  version: v1alpha1
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: []
+  storedVersions: []
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: vsphere-provider-manager-role
+rules:
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
+- apiGroups:
+  - vsphereproviderconfig.sigs.k8s.io
+  resources:
+  - vsphereclusterproviderconfigs
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
+- apiGroups:
+  - vsphereproviderconfig.sigs.k8s.io
+  resources:
+  - vspheremachineproviderconfigs
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
+- apiGroups:
+  - cluster.k8s.io
+  resources:
+  - clusters
+  - clusters/status
+  - machines
+  - machines/status
+  - machinesets
+  - machinedeployments
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  - events
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  creationTimestamp: null
+  name: vsphere-provider-manager-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: vsphere-provider-manager-role
+subjects:
+- kind: ServiceAccount
+  name: default
+  namespace: vsphere-provider-system
+---
+apiVersion: v1
+data:
+  vsphere_tmp: ${TARGET_VM_SSH}
+  vsphere_tmp.pub: ${TARGET_VM_SSH_PUB}
+kind: Secret
+metadata:
+  name: vsphere-provider-machine-sshkeys-546f6942dk
+  namespace: vsphere-provider-system
+type: Opaque
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    control-plane: controller-manager
+    controller-tools.k8s.io: "1.0"
+  name: vsphere-provider-controller-manager-service
+  namespace: vsphere-provider-system
+spec:
+  ports:
+  - port: 443
+  selector:
+    control-plane: controller-manager
+    controller-tools.k8s.io: "1.0"
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    control-plane: controller-manager
+    controller-tools.k8s.io: "1.0"
+  name: vsphere-provider-controller-manager
+  namespace: vsphere-provider-system
+spec:
+  selector:
+    matchLabels:
+      control-plane: controller-manager
+      controller-tools.k8s.io: "1.0"
+  serviceName: vsphere-provider-controller-manager-service
+  template:
+    metadata:
+      labels:
+        control-plane: controller-manager
+        controller-tools.k8s.io: "1.0"
+    spec:
+      containers:
+      - args:
+        - --logtostderr
```

Comment:
fyi, if you want to add more debug logging, add another arg, - --v=<some high number>.  In the future when we grab the logs when the e2e fails, that's what we should use.

Reply:
FWIW, I agree. The `--v=2` is the standard, trailing argument for all K8s-related configs I've seen.

Reply:
set to 6 now